setup.cfg

```diff
@@ -13,3 +13,11 @@ select=
     # E111: Indentation is not a multiple of four
     E114
     # E114: Indentation is not a multiple of four (comment)
+
+[versioneer]
```

Comment:
Can we adopt a convention of putting two blank lines between blocks?  In particular I'm thinking `[coverage:run]` and `[coverage:report]` should have 1 blank line, so more-distinct blocks can have one more line.

Obviously, not a big deal.